core/storage/car/cids_index/cids_index.hpp

```diff
@@ -63,9 +63,9 @@ namespace fc::storage::cids_index {
     return memcmp(&l.key, &r, sizeof(CbCid)) < 0;
   }
 
-  static inline const Row kHeaderV0{
+  inline const Row kHeaderV0{
```

Comment:
Why it was removed from static state, why do we need to instantiate the class to get access to constant that is used in higher levels of program call trace ?

Reply:
[`static` linkage](https://en.cppreference.com/w/cpp/language/storage_duration) != [`static` member](https://en.cppreference.com/w/cpp/language/static)